classifiers/classifier_images.py

```diff
@@ -71,8 +71,8 @@ def fit_zsl(self):
                    
                 # inputv = Variable(self.input)
                 # labelv = Variable(self.label)
-                inputv = self.input.detach()
-                labelv = self.label.detach()
+                inputv = self.input.clone()
```

Comment:
Check if `clone` is required. It might also work without using clone. If it works, we might need to remove some redundant variables.

Reply:
Okay.